resteasy-client/src/main/java/org/jboss/resteasy/client/jaxrs/internal/proxy/ClientInvoker.java

```diff
@@ -142,29 +138,11 @@ protected Object invokeSync(Object[] args) {
 
     protected ClientInvocation createRequest(Object[] args) {
         WebTarget target = this.webTarget;
+        for (int i = 0; i < processors.length; i++) {
```

Comment:
@jamezp processors.length looks suspicious. If processors can be null then it might blow up here,
OTOH if it cannot be null then why check for null on the next line?